src/rx/rx_flysky_hal.c

```diff
@@ -0,0 +1,96 @@
+#include "project.h"
+
+#if defined(RX_FLYSKY)
+#include "drv_time.h"
+#include "drv_gpio.h"
+#include "rx_flysky.h"
+
+//------------------------------------------------------------------------------
+// This file contains hardware related functions and is kept separate
+// from the rest of the RX code so that the RX code can be developed/used/tested
+// with other projects/frameworks
+
+//------------------------------------------------------------------------------
+// If 'init' is true, this records state info about the bind button pin so we can
+// later debounce and detect state changes. Otherwise it checks the state
+// (performing a debounce check) and returns true if button was depressed
+bool flysky_check_bind_button(bool init) {
```

Comment:
historically we do not use bind buttons. personally i hate them, feels like something that can just be handled so much more convenient via other means, but if are to introduce them, this should be a system for all protocols.
please eliminate and if we want to change the behaviour here, let's revisit in another pr/whatever.

Reply:
Happy to remove bind button support. Removed with e81d403